benchmark/memory/memory.benchmark.cpp

```diff
@@ -0,0 +1,255 @@
+#include <mbgl/benchmark/stub_file_source.hpp>
+#include <mbgl/benchmark/util.hpp>
+
+#include <mbgl/map/map.hpp>
+#include <mbgl/gl/offscreen_view.hpp>
+#include <mbgl/gl/headless_backend.hpp>
+#include <mbgl/util/default_thread_pool.hpp>
+#include <mbgl/util/io.hpp>
+#include <mbgl/util/run_loop.hpp>
+
+#include <algorithm>
+#include <iostream>
+#include <iterator>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <sstream>
+
+#include <stdlib.h>
+#include <unistd.h>
+
+/*
+ * Begin getRSS.c
```

Comment:
Can we move this to `benchmark/src/mbgl/benchmark/getrss.c`? Maybe add the function prototypes to `utils.hpp`.